LuaEngine.pas

```diff
@@ -132,6 +132,11 @@   TLuaEngine = class(TObject)
 
     procedure RegisterFunction(const AName: string; const AFunction: TLuaExternalFunction);
 
+    procedure PushGlobVar(const AName: string; const Value: Boolean); overload;
+    procedure PushGlobVar(const AName: string; const Value: Integer); overload;
+    procedure PushGlobVar(const AName: string; const Value: Double); overload;
+    procedure PushGlobVar(const AName: string; const Value: RawByteString); overload;

```

Comment:
1. Rename to PushGlobalVar
2. Add PushGlobalVar(const AName: string; const Data: Pointer; const DataSize: Integer) - use lua_pushlstring to init variable